### PR TITLE
[docs] Add helper script for registryDockerCfg generation

### PR DESCRIPTION
**Pull request by unreturned**
## Description
Add helper script for `registryDockerCfg` generation.
Beautifier `kubectl exec -ti -n d8-system` to `kubectl -n d8-system exec -ti`.

## Changelog entries

```changes
section: docs
type: chore
summary: Add helper script for `registryDockerCfg` generation.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 1]
```diff
@@ -369,6 +369,19 @@ Use the following `registryDockerCfg` if authentication is required to access De
 
 `registryDockerCfg` must be Base64-encoded.
 
+You can use this helper script to generate `registryDockerCfg`:
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:372]:
```suggestion
You can use the following script to generate `registryDockerCfg`:
```

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 2]
```diff
@@ -370,6 +370,19 @@ Deckhouse поддерживает работу только с Bearer token-с
 
 Итоговое значение для `registryDockerCfg` должно быть также закодировано в Base64.
 
+Вы можете использовать этот скрипт-помощник для генерации `registryDockerCfg`:
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:373]:
```suggestion
Вы можете использовать следующий скрипт для генерации `registryDockerCfg`:
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Failure: cherry-pick commit 3040d6b84910ff295f80f125364c9296a997f5f9 to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/11320825585) for details.

**Comment by z9r5** [comment]
Cherry-picked into release-1.64

**Pull request merged**